osgar/lib/pplanner.py

```diff
@@ -0,0 +1,54 @@
+"""
+Simple Path Planner
+"""
+
+
+def find_path(img, start, finish, verbose=False):
+    max_x, max_y = img.shape
+
+    if not (0 <= start[0] < max_x and 0 <=start[1] < max_y):
+        return None
+
+    if not img[start[1]][start[0]]:
+        return None
+
+    queue = [(start, None)]
+
+    expanded = set()
+    prev = {}
+
+
+    ret = None
+    while len(queue) > 0:
+        head, queue = queue[0], queue[1:]
+        if verbose:
+            print(head)
+        if head[0] in expanded:
+            continue
+        expanded.add(head[0])
+
+        if head[0] in finish:
+            ret = [head[0]]
+            p = head[1]
+            while p is not None:
+                ret.append(p)
+                p = prev[p]
+            ret = list(reversed(ret))
+            break
```

Comment:
```suggestion
            return ret
```
or a break directly after `if` and the rest of the code moved to the end of the function?